### ISSUE TITLE
Publish on PS Gallery

### ISSUE THREAD

**Issue by alagoutte** [benclaussen/NetboxPS#2]
Hi,

Thanks for your module !

There is some plan to publish this module on PS Gallery ? (more easy for install, don't need git...)

**Comment by benclaussen** [comment]
Yes.. but I can't give a timeline at the moment. There are a number of breaking changes in Netbox 2.8 that I have not updated this module for. I've also been learning as I go and publishing to repositories is on that list 🙂

**Comment by alagoutte** [comment]
Hi Ben 

Ok ! I can help if you want...

I found a first issue when use self signed certificate...

May be also need to enhance doc and a how to for use your test.
Do you think add integration test ? With the docker images of netbox ? I will be more complete test 

You can contact me by private if you want

**Comment by benclaussen** [comment]
So this was included in #22, however it seems `System.Web` assembly is not available in .NET Core. This module makes use of it in `Functions\Helpers\BuildNewURI.ps1` to create a URI query string (~Line 80). 

We will need to implement a new function to build this string and properly escape characters without using the `System.Web` class.

**Comment by benclaussen** [comment]
FYI I manually published to PSGallery just now
